Src/WitsmlExplorer.Api/Workers/Copy/CopyLogWithParentWorker.cs

```diff
@@ -0,0 +1,58 @@
+using System.Threading.Tasks;
+
+using Microsoft.Extensions.Logging;
+
+using WitsmlExplorer.Api.Jobs;
+using WitsmlExplorer.Api.Models;
+using WitsmlExplorer.Api.Services;
+
+namespace WitsmlExplorer.Api.Workers.Copy
+{
+    public class CopyLogWithParentWorker : BaseWorker<CopyLogWithParentJob>, IWorker
+    {
+        private readonly ICopyWellWorker _copyWellWorker;
+        private readonly ICopyWellboreWorker _copyWellboreWorker;
+
+        private readonly ICopyLogWorker _copyLogWorker;
+
+        public CopyLogWithParentWorker(
+            ILogger<CopyLogWithParentJob> logger,
+            IWitsmlClientProvider witsmlClientProvider,
+            ICopyWellWorker copyWellWorker,
+            ICopyWellboreWorker copyWellboreWorker,
+            ICopyLogWorker copyLogWorker)
+            : base(witsmlClientProvider, logger)
+        {
+            _copyWellWorker = copyWellWorker;
+            _copyWellboreWorker = copyWellboreWorker;
+            _copyLogWorker = copyLogWorker;
+        }
+
+        public JobType JobType => JobType.CopyLogWithParent;
+
+        public override async Task<(WorkerResult WorkerResult, RefreshAction RefreshAction)> Execute(CopyLogWithParentJob job)
+        {
+            if (job.CopyWellJob != null)
+            {
+                (WorkerResult result, RefreshAction) wellResult = await _copyWellWorker.Execute(job.CopyWellJob);
+
+                if (!wellResult.result.IsSuccess)
+                {
+                    return wellResult;
+                }
+            }
+
+            if (job.CopyWellboreJob != null)
+            {
+                (WorkerResult result, RefreshAction) wellboreResult = await _copyWellboreWorker.Execute(job.CopyWellboreJob);
+
+                if (!wellboreResult.result.IsSuccess)
+                {
+                    return wellboreResult;
+                }
+            }
+
+            return await _copyLogWorker.Execute(new() { Source = job.Source, Target = job.Target });
```

Comment:
`_copyLogWorker` returns a refreshAction that refreshes the logs on a given wellbore. This is insufficient when the worker copies over the well or the wellbore. I think the following would ensure correct refreshing:
`        public override async Task<(WorkerResult WorkerResult, RefreshAction RefreshAction)> Execute(CopyLogWithParentJob job)
        {
            RefreshAction refreshAction = null;
            if (job.CopyWellJob != null)
            {
                (WorkerResult result, RefreshAction refreshWell) wellResult = await _copyWellWorker.Execute(job.CopyWellJob);
                refreshAction = wellResult.refreshWell;
                if (!wellResult.result.IsSuccess)
                {
                    return wellResult;
                }
            }

            if (job.CopyWellboreJob != null)
            {
                (WorkerResult result, RefreshAction refreshWellbore) wellboreResult = await _copyWellboreWorker.Execute(job.CopyWellboreJob);
                refreshAction ??= wellboreResult.refreshWellbore;
                if (!wellboreResult.result.IsSuccess)
                {
                    return wellboreResult;
                }
            }

            (WorkerResult result, RefreshAction refreshLogs) = await _copyLogWorker.Execute(new() { Source = job.Source, Target = job.Target });
            refreshAction ??= refreshLogs;
            return (result, refreshAction);
        }`
       
BTW there is currently a bug in frontend regarding refreshing, that can be fixed by adding `wells` to the dependency list in RefreshHandler.tsx on line 99.